packages/toc/src/generators/notebook/tagstool/index.tsx

```diff
@@ -214,6 +242,25 @@ class TagsToolComponent extends React.Component<IProperties, IState> {
         </span>
       );
     }
+    let command;
+
+    if (this.state.selected.length === 0) {
+      command = (
+        <span className={'toc-filter-button-na'}>
+          Select All Cells With Current Tags
```

Comment:
Thanks for working on this @NPetz !

Could you please add internationalization to this ?

See for examples: https://github.com/jupyterlab/jupyterlab/blob/f0790b3ffd187b7dc923156b8ae227855abecefb/packages/toc/src/generators/notebook/toolbar_generator.tsx#L175